src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/solution/optislang_manager.py

```diff
@@ -0,0 +1,76 @@
+# ©2023, ANSYS Inc. Unauthorized use, distribution or duplication is prohibited.
+
+import httpx
+
+from ansys.optislang.core import Optislang
+from pathlib import Path
+from typing import Optional, Union
+
+from ansys.saf.glow.core.instance_manager import AbstractInstanceIdentificationClient
+from ansys.saf.glow.solution import FileReference
+from ansys.saf.glow.solution import InstanceManager
+from ansys.saf.glow.storage.base import AbstractStoragePath
+
+
+PRODUCT_NAME = "optislang"
+SERVICE_NAME = "http"
+
+
+class OptislangManager(InstanceManager[Optislang]):
+    def __init__(
+        self,
+        instance_identification: AbstractInstanceIdentificationClient,
+        method_directory: Optional[AbstractStoragePath] = None,
+    ):
+        super().__init__(instance_identification, method_directory)
+
+    def initialize(
+        self,
+        version: Optional[str] = None,
+        project_path: Optional[Union[FileReference, str]] = None,
+        project_properties_file: Optional[Union[FileReference, str]] = None
+    ):
+        """Initialize and start the optiSLang server."""
+        self.initialize_service(SERVICE_NAME, version)
+
+        if project_path:
+            self._upload_file(
+                project_path,
+                str(Path(self.protected_instance_state_directory_name) / "project.opf"),
+                protected=False
```

Comment:
Do we need this? As we do not do anything in the save_state and load_state, I guess we do not need to copy this file?

Reply:
Removed.